benchmarks/send-recv-core.py

```diff
@@ -293,32 +302,39 @@ def op_completed():
                 blocking_recv(worker, ep, recv_msg)
 
         stop = clock()
-        times.append(stop - start)
+        if i >= args.n_warmup_iter:
+            times.append(stop - start)
 
     if args.delay_progress:
-        while finished[0] != 2 * args.n_iter:
+        while finished[0] != 2 * (args.n_iter + args.n_warmup_iter):
             worker.progress()
 
     if args.cuda_profile:
         xp.cuda.profiler.stop()
 
     assert len(times) == args.n_iter
+    bw_avg = format_bytes(2 * args.n_iter * args.n_bytes / sum(times))
+    bw_med = format_bytes(2 * args.n_bytes / np.median(times))
+    lat_avg = int(sum(times) * 1e9 / (2 * args.n_iter))
+    lat_med = int(np.median(times) * 1e9 / 2)
```

Comment:
Could use `dask.utils.format_time` here instead.

Reply:
We can't have Dask dependencies in UCX-Py, it is used by projects who do not use Dask.

Reply:
The `format_bytes` call needs to go then (since that comes from `dask.utils` AFAICS)

Reply:
You're right, we need to remove that. I'll work on that later today or next week.

Reply:
This is done now in https://github.com/rapidsai/ucx-py/pull/863/commits/8741b275d2c69cec2b27289bfa819cea2301fdff .